apps/editors/views.py

```diff
@@ -481,6 +481,7 @@ def queue_fast_track(request):
 @addons_reviewer_required
 def queue_moderated(request):
     rf = (Review.objects.exclude(Q(addon__isnull=True) |
+                                 Q(addon__is_listed=False) |
                                  Q(reviewflag__isnull=True))
                         .filter(editorreview=1,
                                 # Only show reviews for listed add-ons.
```

Comment:
I'd use the word `public` in the comment instead of `listed` to avoid confusion.